pkgs/development/web/nodejs/v20.nix

```diff
@@ -44,5 +44,10 @@ buildNodejs {
       stripLen = 1;
       hash = "sha256-6y3aEqxNC4iTQEv1oewodJrhOHxjp5xZMq1P1QL94Rg=";
     })
+    # fixes test failure, remove once included in release
+    (fetchpatch {
```

Comment:
```suggestion
    (fetchpatch2 {
```

Reply:
No need. In fact I was told to use fetchpatch instead of fetchpatch2 previously, since the latter requires adding `?full_index=1`